README.md

```diff
@@ -88,31 +88,49 @@ The easiest way to build vcf-validator is using the Docker image provided with t
 
 Executables will be created in the `build/bin` subfolder.
 
+
 ### Linux
 
+The build has been tested on the following distros:
+* Ubuntu
+* Arch
+
 The build has been tested on the following compilers:
 * Clang 3.9 to 5.0
 * GCC 4.8 to 6.0
 
-#### Dependencies
+#### Using install_dependencies.sh (Recommended for normal users)
```

Comment:
Rename this as 'Dependencies', first describing them without the installation instructions, then point to the installation script and drop the instructions for the manual build.